src/zenml/zen_server/routers/users_endpoints.py

```diff
@@ -402,7 +454,8 @@ def email_opt_in_response(
                     email=user_response.email,
                     source="zenml server",
                 )
-
+            user_update.activation_token = user.activation_token
+            user_update.active = user.active
```

Comment:
Not needed, the user update is created locally, not by the client.

```suggestion
```

Reply:
Won't it default active to False as in Update model?

Reply:
I removed it - tests seems unaffected, which is a good sign

Reply:
That's why I suggested that the `is_admin` field in the UserRequest doesn't have a default value (which it still does, unfortunately). Can you please remove it?

Reply:
It should have defaulted to None, but I guess this is also okay because the UserSchema looks for _fields that are set_, instead of fields that are not-None:
```
    def update_user(self, user_update: UserUpdate) -> "UserSchema":
        """Update a `UserSchema` from a `UserUpdate`.

        Args:
            user_update: The `UserUpdate` from which to update the schema.

        Returns:
            The updated `UserSchema`.
        """
        for field, value in user_update.dict(exclude_unset=True).items():
...
```

Reply:
I updated the `UserBase` to have only nullable fields and added a root validator in Request to keep legacy behavior unaffected.